app/services/cosmos/cosmos.utils.ts

```diff
@@ -50,3 +69,129 @@ export function calculateUnbondingDelegationBalanceFromResultEntries(entries: Co
 export function validateAccountAddress(address: string) {
   return /^cosmos1[ac-hj-np-z02-9]{38}$/.test(address)
 }
+
+export function convertDelegationDelegatorReward
+  (reward: DelegationDelegatorReward): CosmosValidatorReward {
+  const { validatorAddress, reward: rewardInput } = reward
+  const coins = rewardInput.map(coin => coin as CosmosCoinResult)
+  return { validator_address: validatorAddress, reward: coins }
+}
+
+function convertDescription
+  (description: Description = {} as Description) {
+  const {
+    moniker = '',
+    identity = '',
+    website = '',
+    details = ''
+  } = description;
+  return { moniker, identity, website, details }
+}
+
+function convertCommissionRates
+  (commissionRates: CommissionRates = {} as CommissionRates) {
+  const { rate = '', maxRate = '', maxChangeRate = '' } = commissionRates
+  return {
+    rate,
+    max_rate: maxRate,
+    max_change_rate: maxChangeRate,
```

Comment:
I think changing the `max_rate` naming to `maxRate` in function callers would be simpler, instead of using a conversion function to convert between underscore and camel case.
Similar for the functions below

Reply:
Does that mean we will also change the field name in the caller like files in app/models/validator and app/models/chain-store and so on?

Reply:
Yes I think in most cases that would make caller code simpler, since we also use camelCase in our code
Correct me if I'm wrong though

Reply:
I see, will hand on renaming ralevent fields

Reply:
The `convertDescription` function reminds me that the `commission` field is also optional.
And these convert functions not only rename the field from camel case to snake case but also provide default string value `''` for optional fields.
Talking about that, do we still need to rename the fields and callers around the app/models?